examples/tx-example.py

```diff
@@ -177,6 +180,18 @@ def get_account_assets():
         print('Asset id = {}, balance = {}'.format(
             asset.asset_id, asset.balance))
 
+@trace 
+def query_transactions(first_time=None, last_time=None, first_height=None, last_height=None):
```

Comment:
According to PEP8 lines should be 80 characters long:
```suggestion
def query_transactions(first_time=None, last_time=None, 
                                     first_height=None, last_height=None):
```